sdk/batch/arm-batch/src/models/parameters.ts

```diff
@@ -20,15 +20,30 @@ export const acceptLanguage: msRest.OperationParameter = {
     }
   }
 };
-export const accountName: msRest.OperationURLParameter = {
+export const accountName0: msRest.OperationURLParameter = {
   parameterPath: "accountName",
   mapper: {
     required: true,
     serializedName: "accountName",
     constraints: {
       MaxLength: 24,
       MinLength: 3,
-      Pattern: /^[-\w\._]+$/
+      Pattern: /^[a-z0-9]+$/
+    },
+    type: {
+      name: "String"
+    }
+  }
+};
+export const accountName1: msRest.OperationURLParameter = {
```

Comment:
I think it would be better to fix the swagger to have one accountName parameter only, i opened a PR with the fix here: https://github.com/Azure/azure-rest-api-specs/pull/11301.

Reply:
That swagger spec is intended to be this way. I will explain it on your PR.

Reply:
I see, I missed the difference in pattern. Thanks for the explanation.